.github/workflows/build.yml

```diff
@@ -0,0 +1,91 @@
+name: Build Project
+
+on:
+    workflow_call:
+
+jobs:
+    build:
+        runs-on: ubuntu-latest
+        outputs:
+            mode: ${{ steps.output.outputs.mode }}
+            type: ${{ steps.output.outputs.type }}
+            version: ${{ steps.output.outputs.version }}
+        steps:
+            - name: Checkout
+              uses: actions/checkout@v2
+
+            - name: Setup Node.js
+              uses: actions/setup-node@v3
+              with:
+                  node-version: 18
+
+            - name: '[⬆] set yarn to stable'
+              run: |
+                  corepack enable
+                  yarn set version stable
+
+            - name: '[ℹ] print versions (node, yarn, git)'
+              run: |
+                  echo "node: $(node -v) – yarn: $(yarn -v) – git: $(git --version)"
+
+            - name: '[⬆] update browserslist'
+              run: npx -y browserslist@latest --update-db
+
+            - name: '[🍱] yarn install'
+              run: yarn install --immutable
+
+            - name: '[🌳] set env variables (e.g. beta/stable for webpack & target dir)'
+              run: |
+                  echo "MODE=$(if ${{ github.ref == 'refs/heads/master' }}; then echo "production"; else echo "development"; fi)" >> $GITHUB_ENV
```

Comment:
Every feature branch is build as dev branch?

Reply:
afaic yes

Reply:
that is correct. Only builds on master should be stable builds.